<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLandUsePlanForm</class>  <!-- Corrected class name -->
 <widget class="QDialog" name="NewLandUsePlanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Land Use Plan</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_plan_id">
       <property name="text">
        <string>Plan ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_plan_id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_plan_name">
       <property name="text">
        <string>Plan Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_plan_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_plan_type">
       <property name="text">
        <string>Plan Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_plan_type">
       <item>
        <property name="text">
         <string>Detailed land use plan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General land use plan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local land use plan</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>NewLandUsePlanForm</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>NewLandUsePlanForm</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
